src/main/java/com/intel/gkl/IntelGKLUtils.java

```diff
@@ -57,16 +56,16 @@
      * @return  true if the native library is supported and loaded, false otherwise
      */
     @Override
-
-
     public synchronized boolean load(File tempDir) {
 
```

Comment:
The input to this function is tempDir .   @SnehalA  do we nee to check for MaxPATH and valid input before calling NativeLibraryLoader or that function takes care of the valid input check?

Reply:
NativeLibraryLoader takes care of this input validation.

Reply:
check for null can be added or handled before passing